test/integration/podtato-head-application/00-install.yaml

```diff
@@ -1,30 +1,13 @@
 apiVersion: lifecycle.keptn.sh/v1beta1
-kind: KeptnApp
+kind: KeptnAppContext
 metadata:
   name: podtato-head
 spec:
   version: "1.3"
-  workloads:
-    - name: podtato-head-left-arm
-      version: 0.1.0
-    - name: podtato-head-left-leg
-      version: 0.1.0
-    - name: podtato-head-entry
-      version: 0.1.0
-    - name: podtato-head-right-arm
-      version: 0.1.0
-    - name: podtato-head-right-leg
-      version: 0.1.0
-    - name: podtato-head-hat
-      version: 0.1.0
-  preDeploymentEvaluations:
```

Comment:
why did you remove evaluations here?

Reply:
wanted to test for just `Tasks` only.
Do you suggest I add it back?

Reply:
since we did a change in the api I would avoid changing the assertion of the tests and keep it as it was, of course adapting if there was any breaking change. you introduced

Reply:
Alright will add it back.